### PR TITLE
Add deployment documentation

### PR DESCRIPTION
**Pull request by mjgiarlo**
This will make it easier for anyone on the team to handle weekly dependency updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,3 +47,43 @@ To bring up a test Solr in a Docker container:
...
+
+### Push updated image
+
+The RIALTO web app is deployed as a container to AWS, and this process is managed by Terraform. The first step of the deployment process is to build, push, and tag an updated image. You can do this manually by following the `Build Docker image` section above and then manually pushing the updated image to Docker Hub:
+
+```
+docker push suldlss/rialto-webapp:latest
+```
+
+In practice, though, merging any commits into the `master` branch—e.g., regular dependency updates—will trigger a CI build which automates building, pushing, and tagging a new image. You can check the [Docker Hub UI](https://hub.docker.com/r/suldlss/rialto-webapp) to see when the image was last updated.
```
Comment (ndushay) [inline README.md]: spaces around your hyphens, pls.

File: README.md [hunk 2]
```diff
@@ -52,3 +47,43 @@ To bring up a test Solr in a Docker container:
...
+
+### Update container definitions in Terraform
+
+Once the new image has been successfully pushed, the container definitions for the RIALTO web app in Terraform must be updated. Create a pull request in the [terraform-aws](https://github.com/sul-dlss/terraform-aws) repo that bumps the value of the `DEPLOYED` environment variable for RIALTO web app in both `staging` and `production` environments. The value chosen is not significant; as long as it changes, a new container will be pulled for the RIALTO web app. That said, our practice has been to bump the date portion of the value to the date the changes flowed into `master` so it is likely best to continue that practice until there is good reason to diverge. See this [pull request](https://github.com/sul-dlss/terraform-aws/pull/471/files) as an example.
```
Comment (ndushay) [inline README.md]: What do you think of changing

```
The value chosen is not significant; as long as it changes, a new container will be pulled for the RIALTO web app. That said, our practice has been to bump the date portion of the value to the date the changes flowed into `master` so it is likely best to continue that practice until there is good reason to diverge.
```

to

```
Our practice is to bump the date portion of the value, which dates the changes flowed into `master`;  the value must change for the new container to be pulled into the RIALTO web app.
```

File: README.md [hunk 3]
```diff
@@ -52,3 +47,43 @@ To bring up a test Solr in a Docker container:
...
+
+Note that a member of the **Operations team** must approve and merge this PR, but the PR does not need to be merged before beginning the next section.
+
+### Deploy to staging environment
+
+Once the Terraform PR in the prior section has been merged—or run this from your branch, no need to wait on `master`!— use Terraform to deploy the RIALTO web app to the staging environment. Note that you will need a [Vault](https://www.vaultproject.io/) account to proceed. Create an [operations tasks](https://github.com/sul-dlss/operations-tasks/) issue to request one if you do not have one already.
```
Comment (ndushay) [inline README.md]: same spaces around hyphens desired pls.

**Comment by peetucket** [comment]
Thanks for writing this up Mike.  I haven't gotten through the whole flow, so cannot confirm, but I like the step by step approach.

File: README.md [hunk 4]
```diff
@@ -52,3 +47,43 @@ To bring up a test Solr in a Docker container:
...
+
+1. Install [Vault](https://www.vaultproject.io/) via your favorite OS package management system. We use Vault to manage secrets and control access to our Terraform infrastructure.
+1. Specify our Vault root URL in an environment variable: `export VAULT_ADDR=https://vault.sul.stanford.edu/`
+1. Get a Vault token: `vault login -method=userpass username=USERNAME password=PASSWORD`
+1. Change into the RIALTO staging terraform directory: `cd {terraform-aws-root-directory}/organizations/staging/rialto`
+1. Ensure to have environment variables set for `AWS_SECRET_ACCESS_KEY` and `AWS_ACCESS_KEY_ID` so that you can be authenticated to AWS.
```
Comment (ndushay) [inline README.md]: s/to/you/

File: README.md (hunk 2 above)
Comment (mjgiarlo) [reply README.md]: :+1:

File: README.md (hunk 3 above)
Comment (mjgiarlo) [reply README.md]: OK

File: README.md (hunk 1 above)
Comment (mjgiarlo) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (mjgiarlo) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (mjgiarlo) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (mjgiarlo) [reply README.md]: done

**Pull request merged**